access-checkout/src/test/java/com/worldpay/access/checkout/ui/AccessCheckoutEditTextTest.kt

```diff
@@ -97,6 +97,15 @@ class AccessCheckoutEditTextTest {
         verify(editTextMock).setHint("some-hint")
     }
 
+    @Test
+    fun `should autofill hint from attribute set`() {
```

Comment:
can we keep the naming in line with other tests to be more consistent?

`should set autofillHints from attribute set`